### ISSUE TITLE
loss = nan?

### ISSUE THREAD

**Issue by WangXin93** [Xiangyu-CAS/Heatmap#8]
Hi，I Just ran your project follow the instruction. I installed all the dependencies, and changed `ann_path` and `img_dir` to my file path. However, I get this `loss = nan`, is that normal? Can you give me some suggestions? Thanks a log! :cake: 
```
Train Iteration: 50	Time 28.197s / 50iters, (0.564)	Data load 16.160s / 50iters, (0.323203)
Learning rate = 4e-05
Loss = nan (ave = nan)

Loss1_1 = nan (ave = nan)
2018-05-25 11:58:20 -----------------------------------------------------------------------------------------------------------------

Train Iteration: 100	Time 23.110s / 50iters, (0.462)	Data load 17.250s / 50iters, (0.344995)
Learning rate = 4e-05
Loss = nan (ave = nan)
```

**Comment by chiragbajaj25** [comment]
Instead of sgd try using Adam optimizer, it worked for me.
